kuka_kr3_support/urdf/kr3r540.xacro

```diff
@@ -1,13 +1,39 @@
 <?xml version="1.0" ?>
-<robot name="kuka_kr3r540" xmlns:xacro="http://wiki.ros.org/xacro">
+<robot name="kr3r540" xmlns:xacro="http://wiki.ros.org/xacro">
```

Comment:
Could you explain why you removed the vendor prefix? There are good reasons for why it's there, but it could be those aren't valid any more, hence my question.

Reply:
@Robotgir please revert this change.